optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/roster/RosterGenerator.java

```diff
@@ -55,12 +55,14 @@
 import org.optaplanner.openshift.employeerostering.shared.tenant.Tenant;
 import org.optaplanner.openshift.employeerostering.shared.tenant.TenantConfiguration;
 
-import static java.util.stream.Collectors.groupingBy;
+import static java.util.stream.Collectors.*;
```

Comment:
Are you sure we want to import all Collectors?

Reply:
for static imports, using  a * is fine normally